### DISCUSSION TITLE
Failed to retrieve token for github copilot

### DISCUSSION THREAD

**Discussion by ghost** [community/community#7600] [Copilot Conversations] [answered]
Here'e the problem:

I have a dual boot system, `windows 10` and `ubuntu 21.04` and on both of them I use `VsCode `and `Intellij `Community edition. Everything was working fine until yesterday. 

When I logged into my ubuntu(fresh installation) today and installed vscode I logged into my github account in order to use copilot. But the problem is the copilot thumbnail in the status bar was gone. I tried installing the copilot extension in Intellij(in ubuntu) as well but I after pasting the confirmation code I got the error :
`Failed to retrieve token for github copilot`.

I then went to windows and the same issue for both VsCode and Intellij. Although the device activation shows `Congratulations, you're all set!`. Intellij shows error: `Failed to retrieve token for github copilot`.

What is the cause of this error? And what the steps to solve this?

This is the screenshot from Intellij

![error_1](https://user-images.githubusercontent.com/50498916/141346145-00b8a21e-4f1a-4e24-af65-5a3d7a36cdee.JPG)

**Comment by ghost** [comment, marked as answer]
As it so happens, I didn't see that the fresh installation messed up my time which in turn interfered with the authentication process. Fixed my system date time and everything went back to normal. I'm gonna keep the solution(?) here so that no one else wastes their time(hopefully).

**Reply by Degobi** [reply]
its work?

**Reply by tusharhow** [reply]
Worked for me, Thanks

**Reply by HarshTambade** [reply]
facing the same issue
getting output failed to get copilot token do help!

**Reply by ghost** [reply]
> facing the same issue getting output failed to get copilot token do help!

Hi techno-king, have you tried to check your system's time? Maybe it's not up-to-date. Please check and see if correcting the time solves the issue

**Comment by murodjon-umar0v** [comment]
i have this problem too. 'User is not Authorized'

**Reply by tusharhow** [reply]
Uninstall Github copilot and vscode. delete all cache. And re-install everything. And login GitHub copilot with GitHub access. It will solve your issue.

**Reply by dennis1232** [reply]
how much time do you wait on the waitlist?

**Reply by tusharhow** [reply]
1 Week. Or wait for release
